Fix bad fieldMapping in change data endpoint. This resulted in 100% CPU
usage for large installs as the data was filtered manually by Python.

https://github.com/buildbot/buildbot/pull/5505
--- buildbot/data/changes.py.orig
+++ buildbot/data/changes.py
@@ -41,7 +41,7 @@ def _fixChange(self, change):
             del change['sourcestampid']
         return change
     fieldMapping = {
-        'changeid': 'changes.id',
+        'changeid': 'changes.changeid',
     }
 
 
